docs/sources/panels/transformations/types-options.md

```diff
@@ -244,6 +281,34 @@ We would then get :
 
 This transformation allows you to extract some key information out of your time series and display them in a convenient way.
 
+## Concatenate fields
+
+> **Note:** This transformation is only available in Grafana 7.3+.
+
+This transformation will combine all fields from all frames into one result.  Consider:
+
+
+Query A:
+
+| Temp    | Uptime  |
+| ------- | ------- |
+| 15.4    | 1230233 |
+
+Query B:
+
+| AQI     | Errors |
+| ------- | ------ |
+| 3.2     | 5      |
+
+
+The result would be:
```

Comment:
The result of what? In the UI, I see three options to select in the **Name** field. What do each of them do? Which one are you using in this example?

Don't forget to mention the option to add a **Label** when you select the **Add a label with the frame name** option.